src/python/pants/backend/native/targets/external_native_library.py

```diff
@@ -0,0 +1,42 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+from pants.base.payload import Payload
+from pants.base.payload_field import PrimitiveField
+from pants.base.validation import assert_list
+from pants.build_graph.target import Target
+
+
+class ExternalNativeLibrary(Target):
```

Comment:
This is a good name for this target at this point in time! It may be worth revisiting later, along with the C/C++ library target names.

Reply:
✅